src/types.ts

```diff
@@ -85,6 +85,7 @@ export interface Activity {
 	state: string;
 	name: string;
 	id: string;
+	url: string;

```

Comment:
Is activity.url is not optional? Every activity does not have url strictly.

Reply:
> Is activity.url is not optional? Every activity does not have url strictly.

Sorry for the late response. Yes, its optional, but it might be useful for a streamer if they want to uses it.

Reply:
Then making it an optional property would be better :)